wagtailio/areweheadlessyet/blocks.py

```diff
@@ -29,7 +29,7 @@ def get_api_representation(self, value, context=None):
         if value is None:
             return None
         return {
-            "date": value.date,
+            "date": value.date.strftime("%d %b %Y"),
```

Comment:
I'd actually suggest not doing this. "formatting" like this should be left down to the frontend / API consumer, rather than being mandated by the API. ISO8601 is easy to parse, and supported by basically everything. Formatting in the frontend allows more customization, and makes it possible to change the representation in different parts of the application. It would also allow using [locale-specific formats](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString).

Reply:
That makes sense. Thanks!